src/util.mjs

```diff
@@ -31,16 +10,6 @@ function arraysEqual(ar1, ar2) {
     return true;
 }
 
-// Check if 2 objects are equal
-function objectsEqual(obj1, obj2) {
```

Comment:
I was so sure that we were using this function, but I couldn't actually find it in the code base... strange! This could come in handy for caching transformed glyphs depending on the set variation coordinates, though, and as it's not adding much in file size (and should actually be tree-shaken if I'm not mistaken) and we already have the working test for it - do we really need to remove it?

Reply:
I just took the opportunity while doing the cleanup of ther other utils.js dead code  (isBrowser/isNode)